ros2param/ros2param/api/__init__.py

```diff
@@ -66,6 +70,26 @@ def load_parameter_file(*, node, node_name, parameter_file, use_wildcard):
             print(msg, file=sys.stderr)
 
 
+def load_parameter_file_atomically(*, node, node_name, parameter_file, use_wildcard):
+    client = AsyncParameterClient(node, node_name)
+    ready = client.wait_for_services(timeout_sec=5.0)
+    if not ready:
+        raise RuntimeError('Wait for service timed out')
+    future = client.load_parameter_file_atomically(parameter_file, use_wildcard)
+    parameters = list(parameter_dict_from_yaml_file(parameter_file, use_wildcard).values())
+    rclpy.spin_until_future_complete(node, future)
+    response = future.result()
+    if response is None:
+        raise RuntimeError('Exception while calling service of node '
+                           f'{node_name}: {future.exception()}')
+
+    if response.result.successful:
```

Comment:
if `response.result.successful` is false, user can know that loading parameter operation has been failed?